web/src/routes/__tests__/PoisPage.spec.tsx

```diff
@@ -1,21 +1,24 @@
 import React from 'react'
+import { ThemeProvider } from 'styled-components'
 
 import { cityContentPath, CityModelBuilder, LanguageModelBuilder, PoiModelBuilder, POIS_ROUTE } from 'api-client'
-import {
-  mockUseLoadFromEndpointWithData,
-  mockUseLoadFromEndpointWithError
-} from 'api-client/src/testing/mockUseLoadFromEndpoint'
+import { mockUseLoadFromEndpointWithData } from 'api-client/src/testing/mockUseLoadFromEndpoint'
 
 import { mockGeolocationSuccess } from '../../__mocks__/geoLocation'
-import { renderRoute } from '../../testing/render'
+import buildConfig from '../../constants/buildConfig'
+import { renderWithRouter } from '../../testing/render'
 import PoisPage from '../PoisPage'
-import { RoutePatterns } from '../index'
 
 jest.mock('api-client', () => ({
   ...jest.requireActual('api-client'),
   useLoadFromEndpoint: jest.fn()
 }))
-jest.mock('react-i18next')
+
+jest.mock('react-i18next', () => ({
+  useTranslation: () => ({
+    t: (key: string) => key
+  })
+}))
```

Comment:
I think I already fixed this but `jest.mock('react-i18next')` is enough (we have a custom mock here)

Reply:
gonna be fixed in your pr so?

Reply:
just wanted to let you know for the next time :)